src/Data/ProgrammesDb/EntityRepository/BroadcastRepository.php

```diff
@@ -93,9 +131,16 @@ private function setEntityTypeFilter(QueryBuilder $qb, string $type, string $bro
 
         if (!is_null($isWebcast)) {
             $qb->andWhere($broadcastAlias . '.isWebcast = :isWebcast')
-                ->setParameter('isWebcast', $isWebcast);
+               ->setParameter('isWebcast', $isWebcast);
         }
 
         return $qb;
     }
+
+    private function getParentProgrammeItem(array $ids): array
```

Comment:
We call this `programmeAncestryGetter` in other Repos, should we stick to that naming convention?